src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -127,4 +130,70 @@ public void appendsAttributesFromInputStream() throws Exception {
         );
     }
 
+    /**
+     * When get on multiple manifests, the first element takes precedence.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void getGetsTheFirstEntryAcrossMultipleManifests() throws Exception {
+        final String name = "Attr";
+        final Manifests mfs = new Manifests();
+        mfs.append(manifestStream("Attr: a\n"));
+        mfs.append(manifestStream("Attr: b\n"));
+        MatcherAssert.assertThat(
+            "The value is from the first manifest",
+            mfs.containsKey(name) && "a".equals(mfs.get(name))
+        );
+    }
+
+    /**
+     * When multiple manifests have been appended, get all gives all the values
+     * of the attribute.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void getAllGetsAListOfAllValuesOfSameKey() throws Exception {
+        final Manifests mfs = new Manifests();
+        mfs.append(manifestStream("Multi-Value-Attr: 1\n"));
+        mfs.append(manifestStream("Multi-Value-Attr: 2\n"));
+        MatcherAssert.assertThat(
+            mfs.getAll("Multi-Value-Attr"),
+            Matchers.contains("1", "2")
+        );
+    }
+
+    /**
+     * When multiple manifests have been appended, get all gives all the values
+     * of two different attributes.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void getAllGetsAListOfAllValues() throws Exception {
+        final Manifests mfs = new Manifests();
+        mfs.append(manifestStream("Attr1: 1a\n"));
```

Comment:
@pinaf What is point of qulice if code reviewer has to look for and enforce a second set of rules in addition to the automated system?